plugins/inputs/jboss/README.md

```diff
@@ -0,0 +1,149 @@
+# JBoss plugin
+
+The JBoss plugin can collect data from JBoss management API.
+
+Plugin currently support JBoss Application server in domain modes:-
+
+- domaincontroller  (Ex http://[jboss-server-ip]:9990/management)
+
+### Configuration:
+
+```toml
+# Read flattened metrics from one or more JBoss HTTP endpoints
+[[inputs.jboss]]
+  ## API endpoint:
+  ##
+  servers = [
+    "http://[jboss-server-ip]:9990/management",
+  ]
+  ## Execution Mode
+	exec_as_domain = false
+  ## Username and password
+  username = ""
+  password = ""
+
+  ## Optional SSL Config
+  # ssl_ca = "/etc/telegraf/ca.pem"
+  # ssl_cert = "/etc/telegraf/cert.pem"
+  # ssl_key = "/etc/telegraf/key.pem"
+  ## Use SSL but skip chain & host verification
+  # insecure_skip_verify = false
+ ## Metric selection
+  metrics =[
+    "jvm",
+    "web_con",
+    "deployment",
+    "database",
+    "jms",
+  ]
+```
+
+Please refer to JBoss management API for full documentation, https://docs.jboss.org/author/display/AS71/The+HTTP+management+API
+
+
+### Measurements & Fields:
```

Comment:
Please check the example readme and update the docs to match: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/EXAMPLE_README.md

Reply:
Sorry I didn't see this comment !!! , now done